.github/workflows/sca.yml

```diff
@@ -86,22 +86,26 @@ jobs:
       - name: Find changed files (for pull request)
         if: ${{ github.event_name == 'pull_request' }}
         run: |
-          git diff origin/$GITHUB_BASE_REF --name-only --diff-filter=ACMRTUXB | grep -E "\.php$" || true
-          echo 'CHANGED_PHP_FILES<<EOF' >> $GITHUB_ENV
-          git diff origin/$GITHUB_BASE_REF --name-only --diff-filter=ACMRTUXB | grep -E "\.php$" || true >> $GITHUB_ENV
-          echo 'EOF' >> $GITHUB_ENV
+          $(git diff origin/$GITHUB_BASE_REF --name-only --diff-filter=ACMRTUXB | grep -E "\.php$")
```

Comment:
so, the CI is failing

Reply:
Didn't anticipate it either. Fixed in latest commit